HEROKU.md

```diff
@@ -15,7 +15,8 @@ After clicking the Heroku button above:
 
 1. Create an account (if you don't have one already).
 2. Give the app a name.
```

Comment:
I know we don't HAVE to have the numbering correct in markdown, but it would be nice since the list isn't that long.